install/installer/pkg/components/spicedb/deployment.go

```diff
@@ -72,6 +80,8 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 									"--datastore-engine=mysql",
 									"--datastore-conn-max-open=100",
 									"--telemetry-endpoint=", // disable telemetry to https://telemetry.authzed.com
+									fmt.Sprintf("--datastore-bootstrap-files=%s", strings.Join(bootstrapFiles, ",")),
+									"--datastore-bootstrap-overwrite=true",
```

Comment:
:question: Does this mean the file overwrites the existing schema in that datastore? :thinking:

Reply:
It does, if it can be overwritten. If there's data that would prevent that then the job fails.